host_guest/README.md

```diff
@@ -9,6 +9,7 @@
 - `OctaAcidsAndGuests/`: Directory containing octa acid (OA and TEMOA) input structures and guest structure files
 - `CB8AndGuests/`: Directory containing CB8 input structure and guest structure files
 - `GenerateInputs.ipynb`: Jupyter notebook using the OpenEye toolkits to generate molecule structure files from the other inputs noted above. NOTE: This is provided for informational purposes; the output files are already available here so it is unnecessary for you to use this notebook.
+- `Reference/`: System files used to run the reference calculations for the main SAMPL challenge.
```

Comment:
Are you putting something in the repo that explains that you ARE doing reference calculations, and what you plan to do?  Otherwise this may be confusing.